frontend/packages/kubevirt-plugin/integration-tests/tests/vm.dashboard.scenario.ts

```diff
@@ -0,0 +1,97 @@
+import { browser, ExpectedConditions as until } from 'protractor';
+import { testName } from '@console/internal-integration-tests/protractor.conf';
+import {
+  addLeakableResource,
+  createResource,
+  removeLeakedResources,
+} from '@console/shared/src/test-utils/utils';
+import { VirtualMachineModel } from '../../src/models';
+import {
+  vmDetailsName,
+  vmDetailsNamespace,
+  vmDetailsNode,
+  vmDetailsIPAddress,
+  vmStatus,
+  vmInventoryNICs,
+  vmInventoryDisks,
+} from '../views/dashboard.view';
+import { getVMManifest, hddDisk, multusNetworkInterface, multusNAD } from './utils/mocks';
+import { VirtualMachine } from './models/virtualMachine';
+import {
+  VM_STATUS,
+  VM_BOOTUP_TIMEOUT_SECS,
+  VM_ACTION,
+  VM_IMPORT_TIMEOUT_SECS,
+  TAB,
+  VM_STOP_TIMEOUT_SECS,
+  NOT_AVAILABLE,
+} from './utils/consts';
+
+describe('Test VM dashboard', () => {
+  const leakedResources = new Set<string>();
+  const testVM = getVMManifest('URL', testName, null, 'foo');
```

Comment:
Nit:
Unless vm name matters, you don't need to specify one and get a generated name
`const testVM  = getVMManifest('URL', testName);`
 It's then not necessary to set the `testVM.metadata.name` on line 43